x-pack/plugins/aiops/public/components/log_categorization/loading_categorization.tsx

```diff
@@ -20,9 +20,10 @@ import { FormattedMessage } from '@kbn/i18n-react';
 
 interface Props {
   onCancel: () => void;
+  disableCancelLoading?: boolean;
```

Comment:
Nit: An alternative to avoid another prop could be to make onCancel optional and set the button to disabled when it's not set.

Reply:
Done in: [#d162011](https://github.com/elastic/kibana/pull/199478/commits/d162011a5397a11864be7ce4fd92d63caa91552b?diff=unified&w=1)